src/commands/deployAzurePipeline/configureCicdPipeline/configurePipeline.ts

```diff
@@ -0,0 +1,47 @@
+import * as vscode from 'vscode';
+import * as k8s from 'vscode-kubernetes-tools-api';
+import { IActionContext } from 'vscode-azureextensionui';
+import { aksClusterNode } from './models/aksClusterNode';
+const CONFIGURE_PIPELINE_COMMAND = 'configure-cicd-pipeline';
+const DEPLOY_TO_AZURE_EXTENSION_ID = 'ms-vscode-deploy-azure.azure-deploy';
+
+export async function configurePipeline(context: IActionContext, target: any): Promise<void> {
+    const deployToAzureExtensionInstalled = await isDeployToAzureExtensionInstalled();
+    if (deployToAzureExtensionInstalled) {
+        const cloudExplorer = await k8s.extension.cloudExplorer.v1;
+        if (cloudExplorer.available) {
+            const clusterTarget = cloudExplorer.api.resolveCommandTarget(target);
+            if (clusterTarget && clusterTarget.cloudName === "Azure" && clusterTarget.nodeType === "resource" && clusterTarget.cloudResource.nodeType === "cluster") {
+                const cluster = new aksClusterNode(clusterTarget);
+                await executeDeployToAzureExtensionInstalled(CONFIGURE_PIPELINE_COMMAND, cluster);
+            } else {
+                vscode.window.showInformationMessage('This command only applies to AKS clusters.');
+            }
+        }
+    } else {
+        vscode.window.showWarningMessage('Please install/enable `Deploy to Azure` extension and start again.');
+        await vscode.commands.executeCommand('extension.open', DEPLOY_TO_AZURE_EXTENSION_ID);
+    }
+}
+
+async function isDeployToAzureExtensionInstalled(): Promise<boolean> {
+    const pipelinesExtension = vscode.extensions.getExtension(DEPLOY_TO_AZURE_EXTENSION_ID);
+    if (!!pipelinesExtension) {
```

Comment:
💡 with this new commit why we are using double bang operator here -> `!!` ~~ `(!!pipelinesExtension)` ?

Reply:
Since ``` pipelinesExtension  ``` is a non-boolean value.
`` !! `` is used to convert it to a boolean one. 

If it's causing a problem, I'll reverse the condition as ws before.